ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.17"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+promotion:
+  to:
+  - name: "4.17"
```

Comment:
do we need also to publish the images to quay?

Reply:
yes, will do in a following pr.

Reply:
:+1: